Readme.md

```diff
@@ -15,6 +15,10 @@ You can find a sample module with a commented walkthrough in this repository. Ch
 Do you want to quickly try out this project and see it in action? Check it out in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution and also see our other useful Orchard Core-related open-source projects!
 
 
+## Demo video
+
+[![Watch the video](https://github.com/Lombiq/Orchard-Base-Theme/blob/27f8592f9b862cb456f93be85af1982f53943f72/docs/assets/images/demo-video-thumbnail.jpg)](https://youtu.be/9DjKxEumoRE)
```

Comment:
Doesn't a relative path work instead of the full GitHub URL? Also, please use a path as we do with docs, as in, _Docs/Assets/Images/DemoVideoThumbnail.jpg_